src/Tribe/PUE/Checker.php

```diff
@@ -637,6 +649,25 @@ private function get_api_message( $info ) {
 			return $message;
 		}
 
+		/**
+         * Whether the plugin is network activated and licensed or not.
```

Comment:
Really Barry?! Really?! Let me add the labels at least....
Faster than the bot...